### PR TITLE
MOBILE: User can view a list of available FoodBags

### PR DESCRIPTION
**Pull request by psnoya**
[MOBILE: User can view a list of available FoodBags](https://www.pivotaltracker.com/story/show/176626859)
### User Story
```
As a recipient,
In order to choose the perfect bag for me
I want to be able to see a list of available bags
```
## Changes proposed in this pull request:
* Created a component folder
* Created DIsplayBagList, Home and Signin components
* Created modules folder
* Created auth.js with autentication
* Created FoodBagService,js file
* Added a bit of styling
* Adding action to tootreducers
* Adding foodbag into initialState
* Removing show action for single foodbag
* Modifying names on foodbag to foodbags
* Fixing various typos
## What I have learned working on this feature:
*Mobile app structure 
*Debugging 
*Connecting BE to FE on mobile
* Plural and Singular names  
## Packages/Gems added
*@react-native-community/async-storage
*@react-native-community/masked-view
*@react-navigation/native
*@react-navigation/stack
*expo
## Screenshots (Client)
Screenshot from 2021-01-29 18-03-18

### CODE REVIEW

File: components/DisplayFoodBagList.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { StyleSheet, View, Text, Dimensions, Image } from 'react-native'
+import { useSelector } from 'react-redux'
+
+const DisplayFoodBagList = ({ navigation, foodbag }) => {
+  const { credentials } = useSelector((state) => state)
+
+  return (
+    <View
+    // style={styles.card}
```
Comment (Ehugo2000) [inline components/DisplayFoodBagList.jsx]: remove comments

**Review by Ehugo2000 — COMMENTED** [review]
Just some small things, but is very noob on mobile...

File: modules/FoodBagService.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+import store from '../state/store/store'
+
+// const API_URL = 'https://git.heroku.com/weca.git'
```
Comment (FlisAnn) [inline modules/FoodBagService.js]: Comment in code

Comment (FlisAnn) [reply modules/FoodBagService.js]: Shouldn't API_URL be active when we merge this to development?

File: state/reducers/rootReducer.js [hunk 3]
```diff
@@ -1,4 +1,14 @@
 const rootReducer = (state, action) => {
+  switch (action.type) {
+    case 'FETCH_FOODBAG_INDEX':
+      return {
+        ...state,
+        foodbags: action.payload,
+      }
+    /*next case goes here*/
```
Comment (FlisAnn) [inline state/reducers/rootReducer.js]: comment

**Pull request merged**